tests/app/templates/home.html

```diff
@@ -12,7 +12,7 @@
     <img src="{% webpack_static 'my-image.png' %}"/>
     <img src="{% webpack_static 'my-image.png' 'APP2'%}"/>
     <div id="react-app"></div>
-    {% render_bundle 'main' 'js' attrs='async charset="UTF-8"'%}
+    {% render_bundle 'main' 'js' attrs='async nonce="{{ CSP_NONCE }}" charset="UTF-8"'%}
```

Comment:
I recon it turns out hard to read, it took me a few minutes to figure how it compiled, but i recon at that time of the day i'm a bit tired :weed:

Did you try using a plain variable ?

         {% render_bundle 'main' 'js' attrs='async nonce="' + CSP_NONCE +'" charset="UTF-8"' %} 

In Jinja you could set CSP_NONCE in a global variable, for django templates a context processor.